src/style.css

```diff
@@ -0,0 +1,100 @@
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body,
+h3 {
+  margin: 0;
+}
+
+.product {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  box-sizing: border-box;
+  border-radius: 5px;
+  color: #000;
+  font-size: 11px;
+  font-weight: 400;
+  padding: 30px 14px;
+}
+
+.product__img {
+  width: 160px;
+  height: 134px;
+  align-self: center;
+}
+
+.product__title {
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 15px;
+  margin: 45px 0 5px;
+}
+
+.product__rating,
+.product__price {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.product__rating {
+  margin: 15px 0 20px;
+}
+
+.product__price {
+  align-items: baseline;
+}
+
+.product__stars {
+  display: flex;
+}
+
+.star {
+  height: 15px;
+  width: 15px;
+  margin-right: 4px;
+  background: url(images/star.svg) no-repeat center;
+}
+
+.product__stars--4 .star:nth-child(-n+4) {
```

Comment:
add rules for all cases, so I can change class, and count of active stars changing accordingly